auth-web/src/components/auth/CreateUserProfileForm.vue

```diff
@@ -28,19 +29,31 @@
       </v-row>
       <!-- Password -->
       <v-row>
-        <v-col cols="12" class="pt-0 pb-0">
+        <v-col cols="12" class="py-0 mb-4">
           <v-text-field
               filled
               label="Password"
               req
               persistent-hint
+              :hint="inputHints.password"
               :rules="passwordRules"
               v-model="password"
               data-test="password"
               type="password"
               :disabled="isLoading"
           >
           </v-text-field>
+          <div
```

Comment:
Probably could simply leverage the v-alert component here?

Reply:
Actually  we dont need that alert there, it was on the old design. it is doing nothing now. 
i will remove that